src/coprocessor/dag/rpn_expr/impl_compare.rs

```diff
@@ -207,6 +207,34 @@ pub fn coalesce<T: Evaluable>(args: &[&Option<T>]) -> Result<Option<T>> {
     Ok(None)
 }
 
+#[rpn_fn(varg)]
+#[inline]
+pub fn compare_in<T: Evaluable + Eq>(args: &[&Option<T>]) -> Result<Option<Int>> {
```

Comment:
Can we add `Eq` to the trait bound of `Evaluable`?

Reply:
Too many other bounds to add if we add one... For example, there are also PartialEq, PartialOrd, etc..